RevenueCatUI/Templates/V2/Variables/VariableHandlerV2.swift

```diff
@@ -306,6 +332,24 @@ extension VariablesV2 {
             )
         case .productStoreProductName:
             return self.productStoreProductName(package: package)
+        case .productGrantsAllAmountCodePerPeriod:
+            return self.productGrantAllAmountCodePerPeriod(package: package, localizations: localizations)
+        case .productGrantsAllAmountNamePerPeriod:
+            return self.productGrantAllAmountNamePerPeriod(package: package,
+                                                           localizations: localizations,
+                                                           virtualCurrencies: virtualCurrencies)
+        case .productGrantsAllAmountCodePerPeriodAbbreviated:
+            return self.productGrantAllAmountCodePerPeriodAbbreviated(package: package, localizations: localizations)
+        case .productGrantsAllAmountNamePerPeriodAbbreviated:
+            return self.productGrantAllAmountNamePerPeriodAbbreviated(package: package,
+                                                                      localizations: localizations,
+                                                                      virtualCurrencies: virtualCurrencies)
+        case .productGrantsAmount:
+            return self.productGrantAmount(package: package, virtualCurrencyCode: variableSubscript ?? "")
+        case .productGrantsName:
+            return self.productGrantName(package: package,
+                                         virtualCurrencyCode: variableSubscript ?? "",
```

Comment:
Nit: although the results would be the same, I might consider only calling `productGrantName` & `productGrantAmount` here if `variableSubscript` is non-nil to avoid the case where we try to look up a virtual currency that doesn't exist.

It doesn't really matter though since `productGrantAmount()` returns an empty string when the VC isn't found, so feel free to ignore 👍